01-Development/01-Developer-Guide/00-Welcome-to-Visual-Studio-and-the-Migrated-Application/Exercice-Application-Structure.md

```diff
@@ -1,10 +1,15 @@
  
 Let us go over the main parts of the Application, using the "Solution Explorer":
-- Find the startup project (Marked **Bold**).
-- Find the Views folder and open the ApplicationMdi.
-- Find and open the Program.cs
-- Find and open the INI file 
-- Find and open the ApplicationPrograms.cs
-- Find and open the ApplicationEntities.cs
-- Find and open the ApplicationCore.cs
-- Go to the base project and look at the Model and the Types Folders 
+1. Find the startup project (Marked **Bold**).
+1. Find the Views folder and open the ApplicationMdi.
+1. Find and open the Program.cs
+1. Find and open the INI file 
+1. Find and open the ApplicationPrograms.cs
+   1. In the ApplicationPrograms find the following programs:
+     1. "Show Orders", what is the class name and what is the project name it is located in?
+     1. What is the class name of the program number 4 and what is the project name it is located in?
+1. Find and open the ApplicationEntities.cs
+   1. What is the name of the 4 table ?

```

Comment:
Please improve wording - instead of "the 4 table"
Table number 4 (consistent to what you've written about programs above)